builtin/providers/aws/structure_test.go

```diff
@@ -481,7 +511,36 @@ func TestFlattenParameters(t *testing.T) {
 	}
 }
 
-func TestFlattenElasticacheParameters(t *testing.T) {
```

Comment:
I don't believe this is actually renaming, instead it's Git's way of displaying it... I think the method was copy-pasted in place and then modified. The method `TestFlattenElasticacheParameters` still exists below